pkg/minishift/openshift/service.go

```diff
@@ -67,38 +107,42 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 		return "", errors.New(fmt.Sprintf("No service with name '%s' defined in namespace '%s'", service, namespace))
 	}
 
-	url := strings.Split(byteArrayToString(cmdOut), "\n")[1] // second element contain actual URL content
+	url := strings.Split(string(cmdOut), "\n")[1] // second element contain actual URL content
 	return urlScheme + url, nil
 }
 
-// Get the available routes to user
-func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
-	var serviceURLs []ServiceURL
+// Get the Service spec like route/nodeport/name in a ServiceSpec structure
```

Comment:
Let's use proper Godoc style docs and start with the function name GetServiceSpecs